src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -441,6 +443,10 @@ identifiers:
 [(14, 7), (14, 10)]  key key
 [(14, 13), (14, 16)]  foo foo
 [(14, 23), (14, 24)]  z z
+[(16, 4), (16, 47)]
```

Comment:
What's this line for? It has no identifier

Reply:
From what I can tell the test prints the value and location for each symbol and I guess for the iteration it produces `undefined` and that got stringified to blank?

Reply:
hmm.. good find, lets try and filter it out before this lands

Reply:
This is happening because of the `formatSymbols` helper in `src/workers/parser/utils/formatSymbols.js` specifically because of [this](https://github.com/devtools-html/debugger.html/blob/master/src/workers/parser/utils/formatSymbols.js#L31)